### DISCUSSION TITLE
Update recommended prior to iOS 18.2 general availability

### DISCUSSION THREAD

**Discussion by Kyle-Reis** [microsoftconnect/intune-app-wrapping-tool-ios#128] [Announcements]
iOS 18.2 (currently in Beta) contains some new features and breaking changes that could allow an organization's data to leave the boundary of the Intune managed identity:

- A new Apple Intelligence feature that will take a screenshot of the application and extract/summarize any text found.
- The new Genmoji feature, which runs out-of-process from the host application, where MAM data protections cannot be applied
- Some breaking changes that prevent the Intune iOS MAM SDK from blocking the Writing Tools feature in certain scenarios.

The latest [20.2.1 release](https://github.com/microsoftconnect/intune-app-wrapping-tool-ios/releases/tag/20.2.1) for Xcode 16 and [19.7.6 release](https://github.com/microsoftconnect/intune-app-wrapping-tool-ios/releases/tag/19.7.6) for Xcode 15 addresses each of these scenarios with support for screen capture blocking, Genmoji blocking, and fixes for the Writing Tools issues.

We recommend that all users of the Intune App Wrapping Tool for iOS rewrap their application with the latest available supported release which corresponds to the Xcode version your application is currently compiled with, and that you push your updated application to production as soon as possible to ensure the fix for this issue is available before iOS 18.2 becomes generally available. If you have any questions, please [file an issue](https://github.com/microsoftconnect/intune-app-wrapping-tool-ios/issues) in this repository or reply directly to this announcement.

**Comment by ajay-x-patel** [comment]
it seems that 20.2.0 also introduced the screen blocking capabilities as we are seeing this in our org already, specifically around the Outlook app. 
Considering how this affects MAM, is there any reason the screen blocking could not be a separate option like it is for Android?
We have a lot of users that actually use screenshots of Outlook to upload to tickets etc but with the new release, we cannot allow screenshots due to the entanglement of "Send data to other apps".

The app configuration would only apply for orgs that enrol devices and not for BYOD that do not enrol into Company Portal in order to download the app configs.

**Comment by Kyle-Reis** [comment]
Hi @ajay-x-patel, we are looking into rolling out a more granular app protection policy setting for screen capture specifically, similar to the one we have on Android. But, given the short timeline we had to respond to the changes in iOS 18.2, we provided the app config as a solution in the interim. Note: admins should configure com.microsoft.intune.mam.screencapturecontrol within MAM app config settings (not MDM app config). MAM app config does NOT require device enrollment. To create a MAM app configuration policy, select "Managed apps" when creating the policy:

![image](https://github.com/user-attachments/assets/c243a57c-dc1c-4fb3-9256-36fbef35bb6e)

**Reply by ajay-x-patel** [reply]
thanks @Kyle-Reis  - that appeases my concerns a whole lot :)
